packages/account/src/Components/reset-trading-password-modal/__tests__/reset-trading-password-modal.spec.tsx

```diff
@@ -40,7 +40,7 @@ const interactWithPasswordField = async (trigger_click = true) => {
 };
 
 describe('<ResetTradingPasswordModal/>', () => {
-    const props = {
+    const props: React.ComponentProps<typeof ResetTradingPasswordModal> = {
```

Comment:
Typescript already detects type since the component in test is imported. Probably not necessary to type it explicitly.

Reply:
Thanks for your revision but we need this type here, if we don't add it TS will throw an error